lib/Controller/ApiController.php

```diff
@@ -405,6 +405,44 @@ public function updateForm(int $id, array $keyValuePairs): DataResponse {
 		return new DataResponse($form->getId());
 	}
 
+	/**
+	 * @NoAdminRequired
+	 *
+	 * Transfer ownership of a form to another user
+	 *
+	 * @param int $formId id of the form to update
+	 * @param string $uid id of the new owner
+	 * @return DataResponse
+	 * @throws OCSBadRequestException
+	 * @throws OCSForbiddenException
+	 */
+	public function ownerTransfer(int $formId, string $uid): DataResponse {
```

Comment:
```suggestion
	public function transferOwner(int $formId, string $uid): DataResponse {
```
I think we should change the name here